### DISCUSSION TITLE
Is there any way to see pomothor in action?

### DISCUSSION THREAD

**Discussion by elcoda** [ozaytsev86/pomothor#28] [General]
Dears,
is a way to view pomothor in action.

In https://pomothor.netlify.app/ I'm unable to sign-on or use github/google login.

I think I'm trying to create something similar... and I could be interested on your project.

Many thanks
Ivan

**Comment by ozaytsev86** [comment]
Hi @elcoda ,

Sorry I was off for some days. Sure you can, I have just restored the project. You can use your github/google account to login. If you need something else drop me a message in discord SickRussel#0645 or use this thread :)

Actually, this is the old version of the project, I refactored it to use Supabase instead of Firebase.

Cheers!

**Comment by elcoda** [comment]
Wonderful.
It's exactly what I was looking for.

Are you still available to publish source-code to install it on-premise?

If so I will try to install it on-premise for my team and , if I can (and I'm able), I want to contribuite.

Thanks

**Reply by ozaytsev86** [reply]
Hey, happy to see that it helped you. Let's talk in discord.
